se_core/include/se/geometry/octree_collision.hpp

```diff
@@ -30,36 +30,32 @@
 #define OCTREE_COLLISION_HPP
 #include "../node.hpp"
 #include "../octree.hpp"
+#include "../utils/eigen_utils.h"
 #include "aabb_collision.hpp"
 
 namespace se {
 namespace geometry {
 enum class collision_status {
-  occupied,
-  unseen,
-  empty
+  occupied, unseen, empty
 };
-
+static std::ostream &operator<<(std::ostream &os, const collision_status &dt) {
+  return os << static_cast<int>(dt);
+}
 /*! \brief Implements a simple state machine to update the collision status.
  * The importance order is given as follows in ascending order: 
  * Empty, Unseen, Occupied.
  * \param previous_status
  * \param new_status 
  */
```

Comment:
Update the documentation with the new function implementation-state machine.

Reply:
done